CONTRIBUTING.md

```diff
@@ -105,7 +105,12 @@ We don't follow any other convention, but if you want to use one, we recommend [
 
 ### Changesets <!-- omit in TOC -->
 
-We use [changesets](https://github.com/Noviny/changesets) to do versioning. What that means is that you need to add a changeset by running `yarn changeset` which contains what packages should be bumped, their associated semver bump types and some markdown which will be inserted into changelogs.
+We use [changesets](https://github.com/Noviny/changesets) to do versioning. What that means is that you need to [add a changeset](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) by running `yarn changeset`. The changeset you create contains what packages should be bumped, their associated semver type and the changelogs.
+
+These changesets are added in the `.changesets` directory at the root of the repository, and should be added in your PR.
+Once there is a [release](#release-on-github-and-npm), these files are automatically removed.
```

Comment:
What do you mean, once you do a release on github or npm? what is removing them?

Reply:
Ok I got it, a PR will be triggered to remove the files. Maybe should we briefly this?